README.md

```diff
@@ -1,5 +1,8 @@
-[Penumbra] is a fully shielded zone for the Cosmos ecosystem, providing private
-trading in any cryptoasset.
+![Penumbra logo](images/penumbra-dark.png#gh-dark-mode-only)
+![Penumbra logo](images/penumbra-light.png#gh-light-mode-only)
```

Comment:
For the light mode it might be better to use the variant with black corona rings, since the light color ones won't show up on a light background.

Reply:
Fixed this.